core/src/main/java/org/infinispan/stats/impl/ClusterCacheStatsImpl.java

```diff
@@ -494,9 +527,9 @@ private DistributedCacheStatsCallable(String cacheName, boolean accurateSize) {
 
          AdvancedCache<Object, Object> remoteCache =
                SecurityActions.getUnwrappedCache(embeddedCacheManager.getCache(cacheName)).getAdvancedCache();
+         CacheMgmtInterceptor stats = SecurityActions.getCacheMgmtInterceptor(remoteCache);
```

Comment:
why going through the `ComponentRegistry` instead of `getFirstInterceptorWhichExtends`?

Reply:
Because I didn't notice `getFirstInterceptorWhichExtends` :)